standalone/ansible/roles/logging/templates/etc/logrotate.d/rails.j2

```diff
@@ -0,0 +1,17 @@
+/home/deploy/apps/*/current/log/*.log {
+  su deploy deploy
+  daily
+  rotate 14
+  missingok
+  notifempty
+  compress
+  delaycompress
+  copytruncate
+  dateext
+  lastaction
+    HOSTNAME=`hostname`
+{% for server in groups.storage %}
+    rsync -a /home/deploy/apps/{{ app_name }}/current/log/*.gz {{ deploy_user }}@{{ server }}:~/logs/$HOSTNAME/
```

Comment:
@prabhanshuguptagit Can you confirm if one server fails on rsyncing, the other rsync should continue to go through and the `lastaction` script does not abort on any failure?

This can be tested by putting up a bogus IP in the `storage` group.

Reply:
I would also suggest to add a log statement for every `rsync`. And also rotate that off through this command itself. `rsync` has a `--log-file` parameter.

Reply:
Confirmed that the `lastaction` script does not abort if one rsync fails.